Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonThemeComboBox.cs

```diff
@@ -50,6 +56,8 @@ public KryptonThemeComboBox()
         #endregion
 
         #region Protected
+        /// <summary>Raises the SelectedIndexChanged event.</summary>
+        /// <param name="e">An EventArgs containing the event data.</param>
         protected override void OnSelectedIndexChanged(EventArgs e)
         {
             ThemeManager.ApplyGlobalTheme(_internalKryptonManager, ThemeManager.ApplyThemeMode(Text));
```

Comment:
This needs to be protected when it being initialised, so that it doesn;t try and set the global to a different value when being initialised

Reply:
Just a thought, can the current style be 'saved' while being initialised

Reply:
No saved per se, but a modification to global code from anywhere can happen.